### ISSUE TITLE
Pressing `F` does not open hint so I can open a tab in new tab

### ISSUE THREAD

**Issue by nikitavoloboev** [brookhong/Surfingkeys#2212]
Pressing `f` opens the hints
<img width="811" alt="image" src="https://github.com/user-attachments/assets/80c45cf2-a0fa-499c-bb30-456e14127287">

But pressing `F` does nothing.

I want to press `F` and see the link hints and on activating them it would open the link in new tab.

**Comment by nikitavoloboev** [comment]
for this part:

> I want to press F and see the link hints and on activating them it would open the link in new tab

I found

<img width="848" alt="image" src="https://github.com/user-attachments/assets/0b9c16e3-5dd3-4e4a-97b7-c0e71ec6f0fd">

but its unclear how to use it. I thought in hint mode here:

<img width="843" alt="image" src="https://github.com/user-attachments/assets/9889c1cf-6759-4b05-921d-2e9a0ef7ab20">

I could just do `shift+<letter>` and if I do that, then it will open the link in new tab and switch to it but it does not do that.

Basically I need:

`f` = hints that let me open tab (in same tab I am in)
`F` = hint that let me open tab in new tab (and not switch to tab after)

and 

`f` = hints (same as `f` now but if I type in the hints in capital letters, it will open the tab in new tab and focus in on that tab)

That would be ideal for me.

Thanks a lot.
